storage/tianmu/core/tianmu_attr.cpp

```diff
@@ -610,27 +613,27 @@ types::BString TianmuAttr::DecodeValue_S(int64_t code) {
   common::ColumnType a_type = TypeName();
   if (ATI::IsIntegerType(a_type)) {
     types::TianmuNum tianmu_n(code, -1, false, a_type);
-    types::BString local_rcb = tianmu_n.ToBString();
-    local_rcb.MakePersistent();
-    return local_rcb;
+    types::BString local_tianmu_b = tianmu_n.ToBString();
```

Comment:
why we do not decleare this var `types::BString local_tianmu_b` at the begiing of this part. Every casese re-define the var. and why do we use `xxx_b`, what the meaning of `_b` is.

Reply:
> 

Yeah, you are right, I'll move this var to beginning of this funciton and change the var name to `_bs`, short for `BString`.